### PR TITLE
fix dupe barell, hopper and dolly

### PR DESCRIPTION
**Pull request by Kotl-EV**
if you put an object in a barrel, pick it up with a dolly and quickly put it and put it on the funnel, then the object will be copied

### CODE REVIEW

File: src/main/java/mcp/mobius/betterbarrels/common/blocks/TileEntityBarrel.java [hunk 1]
```diff
@@ -669,9 +669,10 @@ public ItemStack getStackInSlot(int islot) {
 	public ItemStack decrStackSize(int islot, int quantity) {
 		TileEntity ent = this.worldObj.getTileEntity(this.xCoord, this.yCoord - 1, this.zCoord);
 		ItemStack stack;
-		if (ent instanceof TileEntityHopper) {
+        //TODO Kotl Remove Code fix dupe
+        /*if (ent instanceof TileEntityHopper) {
```
Comment (repo-alt) [inline src/main/java/mcp/mobius/betterbarrels/common/blocks/TileEntityBarrel.java:673]: Why that code was there in the first place? Isn't it there to fix the same issue that storage drawers had?

**Comment by Dream-Master** [comment]
@Kotl-EV ? Any news here ?

File: src/main/java/mcp/mobius/betterbarrels/common/blocks/TileEntityBarrel.java (hunk 1 above)
Comment (Kotl-EV) [reply src/main/java/mcp/mobius/betterbarrels/common/blocks/TileEntityBarrel.java:673]: I don't know why this code is there, but the fact is that with it there is a dupe without it.

**Review by Dream-Master — APPROVED** [review]

**Comment by Dream-Master** [comment]
lets test on zeta

File: src/main/java/mcp/mobius/betterbarrels/common/blocks/TileEntityBarrel.java (hunk 1 above)
Comment (Kotl-EV) [reply src/main/java/mcp/mobius/betterbarrels/common/blocks/TileEntityBarrel.java:673]: removing this code did not cause any negative results

**Pull request merged**